manager/src/main/java/com/redhat/service/bridge/manager/RhoasServiceImpl.java

```diff
@@ -34,8 +44,9 @@ public Topic createTopicAndGrantAccessFor(String topicName, RhoasTopicAccessType
             NewTopicInput newTopicInput = new NewTopicInput()
                     .name(topicName)
                     .settings(new TopicSettings().numPartitions(1));
-
-            return rhoasClient.createTopicAndGrantAccess(newTopicInput, rhoasOpsAccountClientId, accessType).await().atMost(Duration.ofSeconds(rhoasTimeout));
+            return rhoasClient.createTopicAndGrantAccess(newTopicInput, rhoasOpsAccountClientId, accessType)
```

Comment:
@r00ta What happens if the Topic creation has succeeded in the first attempt, but the ACL creation fails? Is the creation of the Topic idempotent e.g. the retry doesn't get an error if the topic already exists?

Reply:
@robpblake If the topic has been created but the ACL creation fails the `createTopicAndGrantAccess` will revert the changes deleting the topic. 

Anyway, the rhoas client api are idempotent in general

Reply:
Thanks @r00ta